doc/WAIMEA.md

```diff
@@ -0,0 +1,158 @@
+Waimea blog
+
+# Waimea: Collective management and compensation for open source communities
+
+Waimea is a new Gov4Git governance mechanism we developed specifically for open-source software development and community management. It is available in [v2.2.0](https://github.com/gov4git/gov4git/releases/tag/v2.2.0) and after.
+
+For context, [Gov4Git](https://gov4git.org) is a platform for augmenting the standard collaborative development workflow — based on issues and pull requests (in GitHub parlance) — with a governance and economics __mechanisms__. From a software perspective, Gov4Git provides an SDK for building mechanisms, as well as a runtime and an app for deploying mechanisms to projects on GitHub, or other similar systems.
+
+## Governance and open source
+
+Every collaborative project has governance. It is the set of rules and processes — be it explicit or implicit, ad-hoc or premeditated, written or unwritten — that determine __prioritization__ of concerns, __decision-making__ on proposals, and __attribution__ of credit to individual collaborators.
+
+At first glance, governance dictates how participants collaborate, and how they are compensated for their effort. 
+
+But, in fact, it is more than that. 
+
+Governance — indirectly — also determines whether participants are willing to collaborate, for how long, and to what degree of involvement. In this regard, collaborators judge the governance system itself to determine if it is able to nurture the type of community that they want to be a member of. They judge if the rules of governance reflect their values and whether their contributions will be acknowledged fairly.
+
+Software development that happens within the context of a business entity — such as startups, corporations, or foundations — inherits the governance (also known as "management") mechanisms and their enforcement from the business.
+
+On the other hand open source — in its purest and most popular form — is practiced outside a business or organization. As a result, open source often defaults to a simple ad-hoc system of __governance, which is typically influenced by the available tools for enforcement, rather than the needs of the community__. For instance, most open source projects on GitHub are essentially forced to adopt a non-nuanced, two-tier hierarchical system of maintainers and contributors.
+
+We believe that this inattention to the governance needs of open source communities is the main cause of well-known problems in open source — lack of __sustainable__ maintainership and support, lack of __funding__, __supply chain vulnerabilities__, as well as __community fragmentation__ upon commercialization or institutionalization of impactful projects.
+
+We think that open source communities should have both tools and standard templates of governance, much like startup companies in the tech industry have tools and templates for all matters of business — management, finance, compensation, bookkeeping, HR, and so on.
```

Comment:
would change "much like startup companies in the tech industry" to "much like tech companies (from startups to large corporations)"